### PR TITLE
U16: Create tiles/background transition

### PR DESCRIPTION
**Pull request by Croxsy**
Added new Background color and tiles where they kind of melt together

### CODE REVIEW

File: public/assets/tiles/tile8.png
Comment (Patertuck) [inline public/assets/tiles/tile8.png:1]: Would keep the rotation of this. Else we need to change up the logic

**Review by Patertuck — CHANGES_REQUESTED** [review]
Beautiful, thank you!
Only 1 minor thing that needs to be done to not introduce problems for the code behind

**Review by C0DECYCLE — APPROVED** [review]
LGTM

**Review by Patertuck — APPROVED** [review]
Looks beautiful, thanks!

**Pull request merged**